src/metadata.h

```diff
@@ -380,7 +382,9 @@ namespace librealsense
         uint32_t    exposure_roi_top;
         uint32_t    exposure_roi_bottom;
         uint32_t    preset;
-        uint32_t    laserPowerMode;
+        uint8_t     emitterMode;
```

Comment:
This is API breaking change and should be accompanied with parsing according to the metadata table versioning:
> md_header   header;
        uint32_t    version;	        uint32_t    **version**;